pkgs/development/python-modules/d2to1/default.nix

```diff
@@ -0,0 +1,24 @@
+{ buildPythonPackage
+, lib
+, fetchPypi
+, nose
+}:
+buildPythonPackage rec {
+  pname = "d2to1";
+  name = "${pname}-${version}";
```

Comment:
The `name` attribute is added by `buildPython*` and should therefore be removed.